profiles/sink/Sink_HDMIIN.yaml

```diff
@@ -18,3 +18,8 @@ dsHdmiIn:
   gameFeatures:
     feature: "allm"
     count: 1
+  # HDMI_COMPATIBILITY_VERSION_14 = 0, /*!< Hdmi Compatibility Version 1.4 */
+  # HDMI_COMPATIBILITY_VERSION_20,     /*!< Hdmi Compatibility Version 2.0 */
```

Comment:
Numbering should be in hex format. add like the below one

(avoided # here in review comment as it is bolding the words. use the below format for comments)
  HDMI_COMPATIBILITY_VERSION_14        = 0x0, /*!< Hdmi Compatibility Version 1.4 */
  HDMI_COMPATIBILITY_VERSION_20,       = 0x01    /*!< Hdmi Compatibility Version 2.0 */
  HDMI_COMPATIBILITY_VERSION_21,       = 0x02   /*!< Hdmi Compatibility Version 2.1 */
  HDMI_COMPATIBILITY_VERSION_MAX    = 0x03  /*!< Out of bounds */
HdmiArcPortID: 0x01

Reply:
done